.cirrus.yml

```diff
@@ -595,23 +595,24 @@ task:
         - git clone https://github.com/flutter/tests.git bin/cache/pkg/tests
         - dart --enable-asserts dev/customer_testing/run_tests.dart --skip-on-fetch-failure --skip-template bin/cache/pkg/tests/registry/*.test
 
-    - name: deploy_gallery-macos # linux- and macos- only
-      # Do not add more tasks here. Nothing is currently deployed from master branch, so it is safe to run
-      # even if a test has failed. The behavior of failing dependencies is non-ideal for infra health.
-      # See also: https://github.com/flutter/flutter/pull/49454
-      depends_on:
-        - analyze-linux
-      environment:
```

Comment:
I think you can add `skip: true # https://github.com/flutter/flutter/issues/54016`?

Reply:
Done